src/DataDefinitionsBundle/DependencyInjection/Compiler/CleanerRegistryCompilerPass.php

```diff
@@ -22,12 +21,12 @@ final class CleanerRegistryCompilerPass extends RegisterSimpleRegistryTypePass
 {
     public const CLEANER_TAG = 'data_definitions.cleaner';
 
-    public function __construct()
-    {
+    public function __construct(
+        ) {
```

Comment:
@dpfaffenbauer This seems to be a weird CS rule. Empty constructor should not be split across two lines and with that indentation IMO. WDYT?

Reply:
empty constructor per-se should not exist. 

We use the same CS rule in CoreShop. I actually only added it for the License headers :D

Reply:
Alright, then that's fine with me. :)